spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+  describe '#register' do
+    describe 'registration payload is invalid' do
+      let(:request_params) { { user: FactoryBot.attributes_for(:user).slice(:full_name, :phone_number) } }
+      it 'responds with 400' do
+        post :register, params: request_params
+
+        expect(response.status).to eq(400)
+      end
+    end
+
+    describe 'registration payload is valid' do
+      let(:facility) { FactoryBot.create(:facility) }
+      let(:user) do
```

Comment:
I'd call this `user_params` since it isn't actually a user.

Reply:
Updated.